README.md

```diff
@@ -29,7 +29,7 @@ The Conventional Precommit Linter is a tool designed to ensure commit messages f
 ### Commit Message Structure
 Commit messages are validated against the following format:
 ```
-<type>(<optional-scope>): <summary>
+<type>(<optional-scope>)<!>: <summary>
```

Comment:
@kumekay Should this be included in our documentation? It's not present in the example message in our code. My suggestion is to omit this detail, as it is also not mentioned in the main example message on [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).